frontend/app/services/activity-service.js

```diff
@@ -26,19 +26,23 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-module.exports = function(HALAPIResource, $http, PathHelper){
+module.exports = function(
+  HALAPIResource,
+  $http,
+  I18n,
+  NotificationsService,
+  ConfigurationService,
+  $timeout,
+  PathHelper){
 
   var ActivityService = {
-    createComment: function(workPackage, activities, descending, comment) {
-      var options = {
-        ajax: {
-          method: "POST",
-          data: JSON.stringify({ comment: comment }),
-          contentType: "application/json; charset=utf-8"
-        }
-      };
-
-      return workPackage.links.addComment.fetch(options);
+    createComment: function(workPackage, comment, notify) {
+      return $http({
+        url: URI(workPackage.links.addComment.url()).addSearch('notify', notify).toString(),
```

Comment:
I can define it like this in `openproject-app.js`:

```
// globals
openprojectApp.constant('URI', window.URI);
```

But I'm not yet convinced that this will please the hound.